README.md

```diff
@@ -26,26 +27,28 @@ For a tighter feedback loop, run individual projects or files, and watch, with `
 ## How do I re-index content from CAPI out to Feast?
 
 1. Set up for local operations, as above
-2. Run `npm run commandline-reindex -- [--composer-id 1234567] [--capi-uri path/to/article/in/capi] [--recipe-uid 0551534c8d93e8da7bb70553b10fa0d0f62534a3]`
+2. Run `npm run commandline-reindex -- [--composer-id 1234567] [--capi-uri path/to/article/in/capi] [--recipeUid 0551534c8d93e8da7bb70553b10fa0d0f62534a3]`
```

Comment:
This one line change is actually for the PR others are auto linting i think.

Reply:
If I'm reading [this](https://github.com/guardian/recipes-backend/blob/07dae096fe7413f729bd5804f1b223c042843083/lambda/recipes-responder/src/commandline-reindex.ts#L86) correctly, all of these parameter values are camel case aren't they?

Reply:
oh actually yes, others are also camel case. since I haven't ran any these arguments hence skipped checking them at the first place. Do you think we can amend these as well in this PR? 
Or, I hope it doesn't mean that letter should be camel case after single hyphen? like `--composer-id` should be passed as `composerId` and I have not checked it?

Reply:
Honestly I haven't dug into the details. Maybe `snake-case` params are automagically converted into `camelCase` somewhere and so it doesn't matter. The important thing is that the README is an accurate reflection of what we need to do. It just struck me as odd that we'd be illustrating a mixture of techniques as it means users will potentially have to refer to the README every time they want to use the script to remind themselves which param should be in which format - if it actually makes a difference, which I assume it does or there would be no need to make this change at all.

Reply:
OK, so the news is not good 😞 

Running this test script proves that there is no magic, and all the arge in this case are in fact camelCase (no idea how I missed that! sorry!)

```javascript
#!/usr/bin/env node

const { parseArgs } = require("node:util");

const {values} = parseArgs({
    options: {
        help: {
            type: 'boolean',
            short: 'h'
        },
        testOne: {
            type: 'string',
            short: 't'
        },
        testTwo: {
            type: 'string',
        }
    }
});

console.log(values);
```

```
$ node testargs.js --help --testOne Hello --testTwo 'everybody here'
[Object: null prototype] {
  help: true,
  testOne: 'Hello',
  testTwo: 'everybody here'
}
```

```
node testargs.js --help --test-one Hello --testTwo 'everybody here'
node:internal/util/parse_args/parse_args:98
    throw new ERR_PARSE_ARGS_UNKNOWN_OPTION(
    ^

TypeError [ERR_PARSE_ARGS_UNKNOWN_OPTION]: Unknown option '--test-one'
    at new NodeError (node:internal/errors:405:5)
    at checkOptionUsage (node:internal/util/parse_args/parse_args:98:11)
    at node:internal/util/parse_args/parse_args:360:9
    at Array.forEach (<anonymous>)
    at parseArgs (node:internal/util/parse_args/parse_args:357:3)
    at Object.<anonymous> (/Users/andy_gallagher/testargs.js:5:18)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12) {
  code: 'ERR_PARSE_ARGS_UNKNOWN_OPTION'
}
```

Behaviour is the same on Node v16.20.2, v18.18.2 and v22.6.0.

@Divs-B please could you update the other arguments in the README? Thanks a lot!

Reply:
Thats a good finding to confirm on this, thankyou 👍 . 
I will update Readme for other `args` too.